docs/src/styles/custom-css.css

```diff
@@ -73,6 +88,10 @@
   align-items: center;
   margin-top: 0;
   background-color: var(--sl-color-accent-high);
+
+  &.action {
```

Comment:
In light mode this button has black font-color while hover on it.

<img width="441" alt="Bildschirmfoto 2024-03-05 um 18 15 32" src="https://github.com/tomalaforge/angular-challenges/assets/46655156/9d46d04c-cc88-43da-b81d-d6d7cd991391">